### DISCUSSION TITLE
Magento 2 - Collapsible Refinementlist

### DISCUSSION THREAD

**Discussion by mpikounis** [algolia/instantsearch#5007] [Q&A]
Hi all,

I am new to Algolia and I've been customizing it to fit our Magento 2 store but I've run into an issue that it giving me a hard time. I want the refinement list widget in our instantsearch page to be collapsible because we have quite a few attributes the user can filter with and the way it is presented now there is a lot of scrolling required. 

I've seen some discussions that mention a collapsible attribute for refinementlist but I suspect this is no longer supported. What is the recommended way of implementing this (accordion like) functionality?

Thanks,
Michael

ps. apologies if I am offtopic

**Comment by Haroenv** [comment]
I'm unfortunately not familiar with which options the magento plugin uses, but the refinementList itself is indeed no longer collapsible. What replaced it is the panel component, which has a collapsible option. You can find more info here: https://www.algolia.com/doc/api-reference/widgets/panel/js/

**Reply by mpikounis** [reply]
Thank you for you reply. It's a shame that such basic functionality is no longer available in the refinementlist. I am sure most ecommerce sites would want their refinement list collapsible and with panels it just gets unnecessarily complicated to implement. I will try following the example provided.
